ui/perfherder/helpers.js

```diff
@@ -527,28 +529,63 @@ export const getTitle = (alertSummary) => {
   }
 
   if (alertsInSummary.length > 1) {
-    title = `${Math.min(
+    const maxMagnitude = Math.max(
       ...alertsInSummary.map((alert) => alert.amount_pct),
-    )} - ${Math.max(...alertsInSummary.map((alert) => alert.amount_pct))}%`;
+    );
+    const minMagnitude = Math.min(
+      ...alertsInSummary.map((alert) => alert.amount_pct),
+    );
+    maxMagnitudeAlert = alertsInSummary.find(
+      (alert) => alert.amount_pct === maxMagnitude,
+    );
+    minMagnitudeAlert = alertsInSummary.find(
+      (alert) => alert.amount_pct === minMagnitude,
+    );
+    title = `${maxMagnitude} - ${minMagnitude}%`;
   } else if (alertsInSummary.length === 1) {
     title = `${alertsInSummary[0].amount_pct}%`;
   } else {
     title = 'Empty alert';
   }
 
   // add test info
-  const testInfo = [
-    ...new Set(alertsInSummary.map((a) => getTestName(a.series_signature))),
-  ]
-    .sort()
-    .join(' / ');
+  const tests = [
+    ...new Set(
+      alertsInSummary.map((alert) => getTestName(alert.series_signature)),
+    ),
+  ];
+  let testInfo = tests.sort().join(' / ');
+  console.log('test ', testInfo);
+  console.log('max ', maxMagnitudeAlert);
+  console.log('min ', minMagnitudeAlert);
+  if (maxMagnitudeAlert && minMagnitudeAlert) {
+    testInfo = `${getTestName(
```

Comment:
Found a bug. For a single regression, this will duplicate its name

Reply:
For a single regression `maxMagnitudeAlert` and `minMagnitudeAlert` will have the value of undefined, line 546 which contains condition ` if (alertsInSummary.length > 1)` will make sure of this.

Reply:
The bug will still reproduce if you have a summary with one regression and multiple improvements.